development/README.md

```diff
@@ -282,4 +282,191 @@ $BUILD_SCRIPTS/qgis-dev-install.sh $BUILD_DIR
 
 ## Configure/build/install QGIS in Qt Creator.app
 
-TODO
+### Install QtCreator 4.0 or better
+
+* Grab the installer from https://www.qt.io/download-open-source/
```

Comment:
I just found that it can also be installed from Homebrew

    brew cask install Caskroom/cask/qt-creator

I assume that updates with homebrew as well which I think is quite nice.